packages/ui/src/components/logo/logo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { LogoNameMap } from './logo-name-map'
+
+interface LogoProps {
+  name: keyof typeof LogoNameMap
+  size?: number
+}
+
+const Logo: React.FC<LogoProps> = ({ name, size = 24 }) => {
+  const icon = LogoNameMap[name]
+
+  return (
+    <svg width={size} height={size} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d={icon.path} fill={`#${icon.hex}`} />
```

Comment:
we can add currentColor here to make it respect the theme

Reply:
https://github.com/user-attachments/assets/0343759c-e8f1-4c4b-bdd5-621d330c1629

Yup, that seems to be working, done.